controller/internal/supervisor/iptablesctrl/acls.go

```diff
@@ -21,19 +21,27 @@ const (
 	udpProto    = "udp"
 )
 
-func (i *Instance) cgroupChainRules(appChain string, netChain string, mark string, tcpPorts, udpPorts string, uid string, proxyPort string, proxyPortSetName string) [][]string {
+func (i *Instance) cgroupChainRules(appChain string, netChain string, mark string, tcpPorts, udpPorts string, uid string, proxyPort string, proxyPortSetName string, isHostMode bool) [][]string {
```

Comment:
I wouldn't pass a parameter here as a bool. Caller should determine chains and pass the target chains instead.